packages/manager/modules/nutanix/src/onboarding/routing.js

```diff
@@ -15,6 +20,15 @@ export default /* @ngInject */ ($stateProvider) => {
         ),
     resolve: {
       hideBreadcrumb: () => true,
+      trackClick: /* @ngInject */ (atInternet) => (subPrefixTracking, hit) => {
+        atInternet.trackClick({
+          name: `${PREFIX_TRACKING_NUTANIX}::${subPrefixTracking}::${hit}`,
+          type: 'action',
+        });
+      },
```

Comment:
**nitpick:**  Since it is only used in controller, you can define the function in controller itself.